src/components/calcite-input/calcite-input.tsx

```diff
@@ -70,6 +71,9 @@ export class CalciteInput {
     this.setDisabledAction();
   }
 
+  /** for number values, displays the locale's group separator */
+  @Prop({ reflect: true }) displayGroupSeparator = false;
```

Comment:
This prop isn't used for styling, right? If not, I think we can disable reflecting.